Updates to the fb-delete

alexjyong opened the pull request:
Love your scripts! I added in some improvements to it:

- Debug mode (right now, just logs more and stops from deleting.)
- command line args (right now, only takes debug, going to add in support for the prompts soon)
- Fixed some bugs that would occur when deleting items from the current year AND month.
- Script now respects the load more button (it was just skipping it at first, leaving items intact that needed to be deleted.
- headless mode ability
- Can select which months for all the years to delete from.

alexjyong committed:
added support to click the "load more events" from a given month ability.

alexjyong committed:
Add files via upload

deleting a function i left in that i don't need anymore.

alexjyong committed:
Add files via upload

Added in some tweaks to the delete function. It now respects the 'Load more' button, so it deletes everything.

alexjyong committed:
Add files via upload

Updated prompts; user can now decide which months to delete, and if they want to run the script in headless mode or not.

alexjyong committed:
Add files via upload

Added in debug mode. and getting ready for command line args.

alexjyong committed:
Added the ability for command line options, debug mode, and fixed up some bugs with deleting items from the current month and year.

Commit:
Adding in electron framework. Renaming repo as well.

Commit:
stashing changes.

alexjyong committed:
Update README.md

updating read me

alexjyong committed:
Update README.md

Commit:
Adding into icon data so we can make executables of this guy.

Commit:
Merge branch 'master' of https://github.com/alexjyong/fb-delete

Merge commit.

alexjyong committed:
Update README.md

Commit:
adding in icons for this guy.

Commit:
fixed a bad path in code for the icon.

alexjyong committed:
Update README.md

alexjyong committed:
Update README.md

Commit:
Removing unneeded files from the original code base.

Commit:
Merge branch 'master' of https://github.com/alexjyong/fb-delete

Commit:
Adding in various icon files.

spieglt commented:
@alexjyong Hey, thank you for the PR and sorry I never got back to you about it. For some reason I received no notifications about this or reported issues. It looks like you have a functioning fork on your account so I'm going to leave the overall structure of this repo as it is, but I appreciate you taking it further, and I hope to pull in your bug fixes for "This Month", the current year, and the "Load more" button at some point. I deleted my Facebook account so don't have one to develop with at the moment.

alexjyong commented:
No problem. I'm glad you liked it. I thought it was ingenious how you used
the basic facebook to do a web crawl. It was also an excuse to learn how to
use Electron for making GUIs, which has been on my bucket list for a while
now.

I plan to post this on my website once I get this set up. Don't worry, I'll
ensure you get all due credit. :)  You were the brains behind the main
operation of the code. I just added a bit onto it. :)

On Mon, May 20, 2019 at 4:43 PM spieglt <notifications@github.com> wrote:

> @alexjyong <https://github.com/alexjyong> Hey, thank you for the PR and
> sorry I never got back to you about it. For some reason I received no
> notifications about this or reported issues. It looks like you have a
> functioning fork on your account so I'm going to leave the overall
> structure of this repo as it is, but I appreciate you taking it further,
> and I hope to pull in your bug fixes for "This Month", the current year,
> and the "Load more" button at some point. I deleted my Facebook account so
> don't have one to develop with at the moment.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/spieglt/fb-delete/pull/5?email_source=notifications&email_token=ABJWCVAHCBE4ZVUN7GSDXXDPWMEPJA5CNFSM4FZUOXB2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODV2ARXI#issuecomment-494143709>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABJWCVHADRKDKMPPSHBIVTTPWMEPJANCNFSM4FZUOXBQ>
> .
>

spieglt commented:
Thank you for the praise, but I'm afraid I don't deserve the credit for the mbasic idea: 
http://www.jaruzel.com/blog/How-I-Erased-5000-Facebook-Comments-and-Likes
https://github.com/Jaruzel/DeleteFacebookActivity
Good luck with it!
